.env.sample

```diff
@@ -1,4 +1,6 @@
 SLACK_BOT_TOKEN=
 SLACK_SIGNING_SECRET=
 SLACK_APP_TOKEN=
+SLACK_BOT_ID=
```

Comment:
Per [@aizlewood40's comment](https://github.com/amido/treeumph/pull/2#discussion_r1046279594) should this be an APP_ID rather than a BOT_ID?

Reply:
Ahh missed that comment. It's a good catch, will update it.

Reply:
@velvetkevorkian @aizlewood40 I had a look and I think what I have is fine. The `message` param contains the `user` prop which is the bot's `user_id` and that shouldn't change. I'll update the env var from `SLACK_BOT_ID` to `SLACK_USER_ID`.